art/attacks/evasion/over_the_air_flickering/over_the_air_flickering_pytorch.py

```diff
@@ -0,0 +1,360 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2021
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software
+# and associated documentation files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or
+# substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+This module contains an implementation of the Over-the-Air Adversarial Flickering attack on video recognition networks.
+
+| Paper link: https://arxiv.org/abs/2002.05123
+"""
+
+import logging
+from typing import Optional, TYPE_CHECKING
+
+import numpy as np
+
+from art.config import ART_NUMPY_DTYPE
+from art.estimators.estimator import BaseEstimator
+from art.estimators.classification.classifier import ClassifierMixin, LossGradientsMixin
+from art.attacks.attack import EvasionAttack
+from art.utils import check_and_transform_label_format, get_labels_np_array
+
+if TYPE_CHECKING:
+    # pylint: disable=C0412
+    import torch
+    from art.utils import CLASSIFIER_LOSS_GRADIENTS_TYPE
+
+
+logger = logging.getLogger(__name__)
+
+
+class OverTheAirFlickeringPyTorch(EvasionAttack):
+    """
+    This module contains an implementation of the Over-the-Air Adversarial Flickering attack on video recognition
+    networks.
+
+    | Paper link: https://arxiv.org/abs/2002.05123
+    """
+
+    attack_params = EvasionAttack.attack_params + [
+        "eps_step",
+        "max_iter",
+        "beta_0",
+        "beta_1",
+        "beta_2",
+        "loss_margin",
+        "batch_size",
+        "start_frame_index",
+        "num_frames",
+        "round_samples",
+    ]
+
+    _estimator_requirements = (BaseEstimator, ClassifierMixin, LossGradientsMixin)
+
+    def __init__(
+        self,
+        classifier: "CLASSIFIER_LOSS_GRADIENTS_TYPE",
```

Comment:
I think this should only be a type of PyTorchClassifier

Reply:
I agree.